Automattic/wp-calypso
# Signup: Add eCommerce partners to store options for AB testing

Adds BlueHost and SiteGround as store options of design type step.
We need them in order to enable AB testing and benchmarking of eCommerce partner hosts.

**Testing steps:** 

1. These tests are not available for existing users, so make sure that you are logged out, or use incognito mode for testing.

2. In the `Dev` menu, under `signupStore` select `designTypeWithStore`, and under `signupStoreBenchmarking` one of the following variants: `bluehost`, `bluehostWithWoo`, `siteground`, or `pressable`. 

![tests](https://cloud.githubusercontent.com/assets/1182160/17312367/781cf1dc-5852-11e6-9d8e-2596c0c5dd08.png)

3. Navigate to `/start` and after filling out the survey, select `An online store` option.

Test live: https://calypso.live/?branch=add/signup-ecommerce-partners
